src/blocks/Table.js

```diff
@@ -107,6 +108,7 @@ const Table = ({ table, ...props }) => {
     <TableSection {...props}>
       <Grid style={{ position: 'relative' }}>
         <Content>
+          <Anchor id="open-source-vs-cloud" />
```

Comment:
The Pricing folder contains only the files used for the block with the cards. It's maybe not the best name I admit 🤔 I'll rename it then. 
The Table here is the block with the comparison, but you're right, let's rename it into `ComparativeTable`, it will be better! 🙏